detekt/detekt.bzl

```diff
@@ -43,12 +51,13 @@ detekt = rule(
         # Later we should allow user to customize Detekt binary.
         "_detekt_cli_jar": attr.label(default = "@detekt_cli_jar//file", allow_single_file = True),
         "srcs": attr.label_list(allow_files = True),
-        "config": attr.label(default = None, allow_single_file = True),
+        "config": attr.label(default = None, allow_single_file = True, doc = "Custom Detekt config file (must end with .yml). If not set Detekt will use its default configuration (mind the Detekt version) https://github.com/arturbosch/detekt/blob/master/detekt-cli/src/main/resources/default-detekt-config.yml"),
+        # TODO: Baselines are not fully supported yet due to Detekt relying on absolute paths which doesn't work with Bazel sandboxing.
+        "_baseline": attr.label(default = None, allow_single_file = True),
```

Comment:
How does this private attr get set if it has no default?

Reply:
I guess `None` is valid value? :D I actually want it to be `None` by default

Similar with `config` btw

Reply:
I thought private attrs could not by set through rule invocation, in which case the value is always None.